api/controllers/game/editmode.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (req, res) {
+  const { isRanked } = req.body;
+  const promiseGame = gameService.findGame({ gameId: req.session.game });
+  const promisePlayer = userService.findUser({ userId: req.session.usr });
+  Promise.all([promiseGame, promisePlayer])
+    .then(function changeAndSave(values) {
+      const [game, player] = values;
+
+      const gameUpdates = {isRanked};
```

Comment:
nitpick: prefer spaces around the var when destructuring
```suggestion
      const gameUpdates = { isRanked };
```